src/helpers/helpers.chart.js

```diff
@@ -80,7 +80,7 @@ export function getChartRect(chart, options) {
   const xDim = getChartDimensionByScale(xScale, {min: options.xMin, max: options.xMax, start: x, end: x2});
   x = xDim.start;
   x2 = xDim.end;
-  const yDim = getChartDimensionByScale(yScale, {min: options.yMin, max: options.yMax, start: y, end: y2});
+  const yDim = getChartDimensionByScale(yScale, {min: options.yMin, max: options.yMax, start: y2, end: y});
```

Comment:
I think we should actually use `top, left, bottom, right` of the scale to determine the fallbacks. Because scales can be stacked. And we should consider the reverse option too.
So for x, we'd have start = left, end = right or if reversed start=right, end=left of the scale. 
And for y start = bottom, end = top or if reversed start=top, end=bottom of the scale.

Reply:
OK! I agree. The check on reverse will be done in `getChartDimensionByScale`

Reply:
done, adding test cases with reverse and stacked

Reply:
> I think we should actually use top, left, bottom, right **of the scale**

This part is not done. Should also add a test with stacked scales when this part is done.

Only testing reverse on x-scale, please add tests with y-scale too. Also hide legend etc not needed for the test. (but do not hide scale whose options we are testing against = the scale with `reverse: true` should be visible)

Reply:
By stacked scales I mean this feature:

![image](https://user-images.githubusercontent.com/27971921/151358344-4ed15623-462d-438a-86e2-bd2cf23587a3.png)

Reply:
ah ok. I have never used! let me try

Reply:
have a look now. It should be better